scss/mixins/_utilities.scss

```
@@ -44,6 +47,14 @@
           #{$property}: $value if($enable-important-utilities, !important, null);
         }
       }
+
+      @each $pseudo in $state {
+        .#{$property-class + $infix + $property-class-modifier}-#{$pseudo}:#{$pseudo} {
```

I think we'd want to keep the class the same, and add the pseudo states to it. Thoughts?

```suggestion
        .#{$property-class + $infix + $property-class-modifier}:#{$pseudo} {
```